iroha-lib/model/CMakeLists.txt

```
@@ -0,0 +1,24 @@
+add_library(iroha_lib_model
+    Query.cpp
+    Tx.cpp
+    TxBatch.cpp
+    Query.hpp
+    Tx.hpp
+    TxBatch.hpp
```

I suggest to pair sources with their headers:
```suggestion
    Query.cpp
    Query.hpp
    Tx.cpp
    Tx.hpp
    TxBatch.cpp
    TxBatch.hpp
```

---

Ok.